modules/op-installing-pipelines-operator-in-web-console.adoc

```diff
@@ -41,12 +41,12 @@ The supported profiles are:
 
 .. Select an *Update Channel*.
 
-*** The `pipelines-<version>` channel is the default channel to install the {pipelines-title} Operator. For example, the default channel to install the {pipelines-title} Operator version `1.7` is `pipelines-1.7`.
-*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. 
+*** The `latest` channel enables installation of the most recent stable version of the {pipelines-title} Operator. Currently, it is the default channel for installing and upgrading the {pipelines-title} Operator.
```

Comment:
Do not know if we need to specify `latest` is the default channel for upgrading.
@ppitonak WDYT?